librz/io/p/io_zip.c

```diff
@@ -157,7 +157,7 @@ static int rz_io_zip_slurp_file(RzIOZipFileObj *zfo) {
 	return res;
 }
 
-RzList *rz_io_zip_get_files(char *archivename, ut32 perm, int mode, int rw) {
+RzList *rz_io_zip_get_files(const char *archivename, ut32 perm, int mode, int rw) {
```

Comment:
Can be `archivename` `NULL` here? Please add a check then. Please also fix the indentation after, and convert pattern of `if (zipArch)` into `if (!zipArch)`.

Reply:
no it can't.